dist/scss/wds-components/_toggles.scss

```diff
@@ -0,0 +1,76 @@
+@import '../wds-variables/colors';
+
+.wds-toggle {
+	&__input {
+		display: none;
+	}
+
+	&__label {
+		cursor: pointer;
+		font-size: $wds-typescale-size-base;
+		padding: 7px 5px 7px 52px;
+		position: relative;
+
+		&:empty {
+			padding-left: 45px;
+		}
+
+		&::before, &::after {
+			content: '';
+			margin: 0;
+			outline: 0;
+			position: absolute;
+			top: 50%;
+			transform: translateY(-50%);
+			transition: all .3s ease;
```

Comment:
In general, it's a good habit to specify which properties should be animated. In the future, you could be changing the property in a selected state and by default it'll be animated:
1. it's consuming browser resources
2. this could be unintended behavior. Understanding at that point which properties should be animated and which not is definitely harder than today.